Updated to fix a broken link

brohrer opened the pull request:
Link to Local Deep Kernel Learning for Efficient Non-linear SVM Prediction obsolete.
Updated to
http://proceedings.mlr.press/v28/jose13.html

azurecla commented:
Hi __@micurd__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (micurd). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

brohrer committed:
Updated to fix a broken link

Link to Local Deep Kernel Learning for Efficient Non-linear SVM Prediction obsolete.
Updated to
http://proceedings.mlr.press/v28/jose13.html
